TTD/docs/TTDLiveRecorder.h/interface-ILiveRecorder.md

```diff
@@ -0,0 +1,70 @@
+# Interface TTD::ILiveRecorder
+
+Defined in [TTD/TTDLiveRecorder.h](README.md).
+
+This interface allows a TTD recording client to interact and interfere with the recorder
+in order to control and enhance the recording.
+
+## Overview
+
+Obtaining the interface is described in the header file's [General overview](README.md#general-overview) section.
+
+A working sample that demonstrates the use of this interface can be found on
+[GitHub](https://github.com/microsoft/WinDbg-Samples/tree/master/TTD/LiveRecorderApiSample).
+
+This interface derives from the standard [IUnknown](https://learn.microsoft.com/en-us/windows/win32/api/unknwn/nn-unknwn-iunknown)
+interface to manage object lifetime and access any other interfaces supported by the object, but it doesn't conform to the full COM specification.
```

Comment:
>but it doesn't conform to the full COM specification. [](http://example.com/codeflow?start=93&length=53)

not sure this is necessary. what part of COM are you thinking? we fully conform with IUnknown semantics and I don't think COM requires activatable (registered) classes.

Reply:
well, I suppose we do not use HRESULTs and we do not export DllGetClassObject.

Reply:
And the whole "apartment" thing (which still gives me the giggles every single time).

I reworded. LMK if you have any further suggestions.